Adds new Meetup Organizers Toolkit (#9)
ansible-community/meetup: meetup_toolkit.md

  * [Follow-up email for the venue scout](#follow-up-email-for-the-venue-scout)
  * [Email template for after-venue confirmation](#email-template-for-after-venue-confirmation)
  * [Email template for reaching out Sponsor for the first time](#email-template-for-reaching-out-sponsor-for-the-first-time)	
  * [Follow-up email with the sponsor](#follow-up-email-with-the-sponsor)
  * [Follow up email with the venue sponsor post confirmation](#follow-up-email-with-the-venue-sponsor-post-confirmation)
  * [Follow up email with the food sponsor post confirmation](#follow-up-email-with-the-food-sponsor-post-confirmation)
  * [Follow up email with the Group Support sponsor post confirmation](#follow-up-email-with-the-group-support-sponsor-post-confirmation)
  * [Promotional post templates](#promotional-post-templates)
  * [Post Templates Pre-Event](#post-templates-pre-event)
  * [Post Temaplate during the event](#post-temaplate-duringthe-event)
  * [Post Temaplate after the event](#post-temaplate-after-the-event)
- [Important Links](#important-links)

<!-- tocstop -->

# Gain awareness of Ansible meetups

## What is an Ansible Meetup?

> Ansible meetup is a gathering of a small to medium-sized group of Systems Engineers, DevOps and SREs. Where they talk about their Automation journey focusing on Ansible, it is a place to teach and learn from each other. Meetup provides a platform for upstream and downstream to meet and share ideas.

Reviewer: > Meetup provides a platform

This sounds like you're talking about Meetup.com, but I don't think you are. Since you already covered the ideas around teaching, learning, and sharing, you can delete this sentence entirely.

Author: Updated.

## How to become a member of the meetup?

> One can be a member of an Ansible meetup group by

Reviewer: ```suggestion
One can be a member of an Ansible meetup group by:
```

Add a colon to indicate a list

Author: Updated.

### Attending the meetups

Check the list of current Ansible meetups. If there is already a meetup in your area, join it and register for an upcoming event.

### Presenting at the meetup

Do you want to present at an Ansible meetup? Do you have an idea for a session? Feel free to reach out to the meetup group either by

- Commenting on the meetup page itself.
- Contacting the organizers via meetup.com direct messages.
- Open an issue at the [Github repo of the meetup group](https://github.com/ansible-community/?q=meetup) (if available).
- Share your idea at [#ansible-social](https://matrix.to/#/#social:ansible.com) Matrix room or one of the [regional rooms](https://matrix.to/#/#ansible-regional:matrix.org).

Meetup organizers always look for good presenters to enrich their events and meetup groups. So do not hold back from sharing your ideas.

> ### Hosting a meetup

Reviewer: You've switched from "the meetup" in the first two items to "a meetup" in the second two. I'd use "a meetup" for all four, I think.

Author: Can you please share the exact line/lines? I am not being able to find the exact lines.

> Want to host a meeting at your organization? Then please reach out

Reviewer: ```suggestion
Want to host a meetup at your organization? Then please reach out
```
We're using "meetup" everywhere else...

Author: Updated.

 - by commenting on the meetup page of the local meetup
 - via social media
 - send us an [email](meetup@ansible.com)

### Organizing a meetup

> No Ansible meetup in your area? Send us an [email](meetup@ansible.com) introducing yourself. Be sure to include a link to your profile at meetup.com.

Reviewer: @cybette is this email correct? Shouldn't we use the RH Community Team mail?

Reviewer 2: the email still works (it's receiving meetup invoices and a lot of spam :sweat_smile: ) 

we can use the team email but either way it will be temporary, when the forum is ready we will point them to make requests in the forum.

Author: So let's keep it for now and later we can move right?

Reviewer 2: yes, let's keep it for now. thanks!

We want to be sure that meetup organizers are committed to building a strong local community of Ansible users. After a brief discussion, we will create the Ansible meetup group in your area, appoint you as a co-organizer, and announce your new group to the world.

Secure a date and location for your first meetup! We will help you by spreading the word on social media, reaching out to potential speakers and sponsors in your area, and suggesting new and interesting Ansible content on a regular basis.

## Who can become an organizer?

> Anyone from the wider Ansible community who is committed to run an Ansible meetup group and plan/organize meetups can become an organizer. The fundamental understanding of the Open Source community, its working, and the belief in this ethos is essential to be an efficient community builder. Some knowledge about Ansible is always a plus.

Reviewer: This reads like "Anyone who wants to run a meetup can run a meetup" - it's a bit circular. Why not just say "Anyone!" - it's largely true, so long as they put the work in :)

Likewise, I agree that understanding FOSS is a good idea, but the list of "essentials" could make people fall into the "skills gap" by believing they're not suitable. All we really need is enthusiasm, we can mentor the rest.

Reviewer 2: @GregSutcliffe I get your point, but I think we should still include commitment and understanding FOSS to set some bar, and also expectation that being an organizer is not a trivial task. There have been several instances where people requested a group and we get everything set up (payments made, repos created, even swag sent) and the organizer just runs one event and that's it. Some even none. Just so they can put on LinkedIn or somewhere that they are XXX group organizer.

Reviewer: That's fair, I was hoping the abuse was less blatant. Perhaps it's not about the person but the evidence? On the change I suggested below about venue being more important than speaker, then evidence of already having some of these things can become a pre-requisite for our support. In other words, use these pages to build your meetup, show us it's working, and *then* we'll add you?

## Ansible Community Code of Conduct

> https://docs.ansible.com/ansible/latest/community/code_of_conduct.html

Reviewer: A link is not sufficient, we need to be clear that we expect people to have read this and accept it in return for using the Ansible Community brand

Author: Agreed + 100%. Added the Code of Conduct.

# Start planning

> Now you seem ready and confident to run an Ansible meetup group. Amazing, here is what you have to do.

Reviewer: > Now you seem ready and confident to run an Ansible meetup group

Assuming people are "ready and confident" seems quite a bold statement at only 16% of the way through the document. I can see that intimidating some people - can we make it more neutral? Perhaps

```suggestion
Planning is key to a successful event, so lets go through some of the things you'll want to think about.
```

Author: Updated.

## Mode of the meetup

There are two main ways to conduct a meetup, Online and In-person.

### Online Meetup

The following is a checklist for having an online meetup

    - Platform to host the meetup
    - Fix a date and time
    - Speaker outreach
    - Social media promotion plans
    - Platform to have a conversation with the attendees, speakers, and organizers

An online meetup allows you to connect to a broader audience living in different time zones along with the regular meetup group members.  The organizers may want to promote the event on the social media platforms of the time zones coinciding with the event timing to extend the reach of the group.

### In Person meetup

The following is a checklist for having a in person meetup

    - Venue to host the meetup
    - Fix a date and time for the meetup
    - Speaker outreach
    - Sponsorship Scout
    - Social media promotion plans
    - Platform to have a conversation with the attendees, speakers, and organizers

## Finding speaker

> Selecting the topics forms the core of any meetup group. Before looking for speakers, if the organizing pays heed to these points, that will be helpful.

Reviewer: ```suggestion
Selecting the topics forms the core of any meetup group. Before looking for speakers, if the organizer pays heed to these points, that will be helpful.
```

Author: Updated.

      1. Social media, email
      2. Community events
      3. Github contributions
      4. Ansible Community Communication platform

While reaching out to the speakers, one has to be careful to refrain from spamming people to give talks at the meetups. If you know the person or someone who can introduce them, that would be wonderful. And if not, a personal touch to the message is necessary for the person to know you, your intention, and your ideas, so they can make a reasoned decision about accepting or declining the invite. Here are some template messages which might be helpful in this process.

## Planning tools

In addition to the meetup.com group, we can create a repo for your meetup group on https://github.com/ansible-community/

You can use the repo to:

1. Get talk ideas and proposals for upcoming meetups from your meetup group
2. Upload and share presentation slides / code samples from speakers
3. Create an issue per event to gather feedback from attendees

## Session ideas

1. What is Ansible?
2. Ansible 101
3. Writing your first playbook
4. How to contribute to Ansible?
5. From Playbook to Rulebook (presentaion for EDA)
6. Ansible Ecosystem

## Venue Scout

> Any in-person event can not happen without a venue. After confirming the speaker(s), a venue is the next most important thing to secure. There are some templates in the email template section which might be helpful in the journey of venue scout.

Reviewer: Is it really more important to have speakers than a venue? How many meetups grew out of regular bar chats? (Hint, it's more than zero :grin:)

I would put venue first. Finding regular speakers is hard, and building a local community of enthusiasts is more important.

Author: Agreed. "After confirming the speaker(s), a venue is the next most important thing to secure"  removing this line.

## Sponsor Scout
 
Meetups don't happen without places to have them, and everyone appreciates food at a meetup. As meetup organizers need sponsors to support their community. There are some email/message templates in the Resources Section below to help you with the sponsor scout.

## Promotion
 
Promotions help us to create awareness about the event. This is an important tool for outreach and the way one can spread news about their event around the world. In the Ansible Community we have certain platforms which one can promote the meetups on. 

### Bullhorn

> The Bullhorn is our newsletter for the Ansible contributor community. Read past issues on the official [Bullhorn’s wiki page](https://github.com/ansible/community/wiki/News#the-bullhorn). Bullhorn gives the reach to spread your message across in the Ansible Community Worldwide.

Reviewer: ```suggestion
The Bullhorn is our newsletter for the Ansible contributor community. Read past issues on the official [Bullhorn’s wiki page](https://github.com/ansible/community/wiki/News#the-bullhorn). The Bullhorn gives you the reach to spread your message across in the Ansible Community worldwide.
```

Author: Updated.

But how to promote your event in Bullhorn?

> Simply hop into [#social:ansible.com (the Ansible Social room on Matrix)](https://matrix.to/#/#social:ansible.com) and leave a message, tagging `@newsbot`. Posts to be made in markdown format. 

Reviewer: ```suggestion
Simply hop into [#social:ansible.com (the Ansible Social room on Matrix)](https://matrix.to/#/#social:ansible.com) and leave a message, tagging `@newsbot`. Posts should be in Markdown format. 
```

Author: Updated.

  
### Social Media Promotion

> Social media plays an important role there. There are some social media post templates in the Resources Section below to help you with the sponsor scout. Ansible has social media presence in the following platforms

Reviewer: `there`? Where? :grin:

```suggestion
Social media plays an important role in raising awareness of your event. There are some social media post templates in the Resources Section below to help you with the sponsor scout. Ansible has social media presence in the following platforms:
```

Author: Updated.

[Linked](https://www.linkedin.com/company/ansible/)
[LinkedIn Ansible User groups](https://www.linkedin.com/groups/4648083/)
[Mastodon](@ansible@fosstodon.org)
[Twitter](https://twitter.com/ansible)

 

> While posting about the event please tag the above mentioned handles for extended reach. There are some social media templates in the Resources Section below to help you with Social Media Promotion. While doing the promotion make sure to tag the willing attendees and of course the sponsor(s) of the event.

Reviewer: `above mentioned handles`

It took me a minute to realise what you meant here. Perhaps we can word that as "Please tag our accounts so we can boost your reach" or something like that?

# During the Event

## Guidelines

Now the Big day is here, the day of the event. And you are all ready for it 🙂. Now here are the Guidelines to follow  during the day of the meetup. These are some steps that, if you follow it, will make your life easier as an organizer.

**Strict Guidelines****

1. Follow the Ansible Community Code of Conduct at all times during the event.
2. Be Inclusive 
3. Take care of the Security concerns.

> 4. Make sure to announce the name of the person of contact in case of CoC violation.

Reviewer: I'm uncomfortable about the idea that we appear to not be offering support for orgainisers with CoC enforcement. It's not something many people will be comfortable with, and we should be clear that **we** are here to help.

@cybette thoughts?

Reviewer 2: we should have a well-defined process for reporting and handling CoC incidents. Currently we have the single email address for reporting (which is also receiving spam but not as much), and a few people who have taken the incidence response workshop. Something to work on for sure.

Reviewer: Assuming we have private spaces for organisers, we can support them there too

Author: This is a practical concern. There are times when the attendees have CoC concern with the meetup organizer so process needs to have several points of contact for complain, and easy and quick process.

5. Mode of communication in case of CoC complaints.
6. Make sure that you have a First Aid kit ready.
7. Ask for permission of the attendees, speakers and volunteers  to be photographed.
8. Ask for permission of the attendees, speakers and volunteers  to be tagged in Social Media.

**Good to follow Guidelines

1. Time Management

> Start and end the event on time. This creates a good practice and 

Reviewer: This is circular - you're saying "good to follow guidelines will make good practice"

```suggestion
Start and end the event on time. Ensure speakers also keep to their time allowance, especially if you have multiple speakers - this is respectful to people's time, especially the latter speakers in the agenda.

Author: Updated.

impression of the meetup group. 

2. Round table introduction

> A round table introduction before the meetup helps us to create a network with the attendees. This helps the organizers to know them, their work and experience level better. Manier times you might get your future speaker, organizer or volunteer for the group from there. Therefore be frank in asking for help. 

Reviewer: ```suggestion
A round table introduction before the meetup helps us to create a network with the attendees. This helps the organizers to know them, their work and experience level better. Many times you might get your future speaker, organizer or volunteer for the group from there. Therefore be frank in asking for help. 
```

Author: Updated.

3. Feedback session

An honest and frank feedback session at the end of the meetup is essential. Encourage the participants to comment on the meetup page or fill up the meetup.com feedback form for giving their feedback.You can ask the following questions in there :

- How was the event ? 
- Was it helpful for the attedees?
- How can the meetup improve?
- What are the topics you would like to hear more about in future sessions?

5. Fix the plan for the next session

It is advisable to plan the next event, the date, venue and brief agenda at the end of the meetup. So the attendees leave  with a definitive date to be booked on their calendar. Also it gives them some idea where the community is heading. Once you make the plan with the attendees they feel more inclusive. Giving their wishes priority  you essentially make them feel a part of the group itself.

6. How to connect with the Ansible Community?

> Encourage the attendees to join the Ansible Community Matrix channel and show them how to join Matrix if they are not there.

Reviewer: Link to Matrix here

Author: https://docs.ansible.com/ansible/latest/community/communication.html#general-channels  -> updated

> Join the Ansible Community Matrix space, find a regional channel they can connect in. Start a local telegram/Signal/WhatsApp group with the attendees.

Reviewer: Link to the Space.

Also, don't recommend Signal, WhatsApp, etc - there's a safeguarding issue because all room participants can see the phone numbers of other participants. I'd suggest a private Matrix room as more appropriate, but I know that's unlikely to work :stuck_out_tongue:

Author: Updated to this -> "You may start a private Matrix room with the attendees to be connected"

## Presentation Templates

There are some presentation templates in the Resources Section below to help you with the presentation ideas and giving those ideas the form of presentation.

> Past Talks 

Reviewer: Missing subheading here?

Author: Yes, I am trying to figure out which format in which the presentations should be uploaded.

You can get ideas for talks from the list of the old talks in here. 

https://github.com/ansible-community/?q=meetup
https://www.youtube.com/@AnsibleCommunity/channels 

# Post Event Activities

## Gathering Feedback 

> It is essential to gather feedback from the participants about the meet at the end of the meetup.It gives us insght on how we, the meetup organizers can improve the meetup content and in any and every way. Presently we use the following platforms to gather feedback :

Reviewer: Why has the organiser now become "we" instead of "you"? The document to this point has been second-person.

Also typo in `insight`...

Author: Updated both.

- meetup.com (Feedback tool)
- Email
- In-person feedback during the meetup
- Relevant Ansible Matrix room 

## Plan for the next meetup 

Decide on the frequency of the meetup you as a group want to have it and then plan for the meetup as soon as possible after the current meetup is over. It gives attendees time in advance to plan for the meetup. Also encourage attendees to share the content for the meetup topic using any of the abovementioned tools and platforms. The meetup groups at liberty to use their preffered platform for communication. In helps to keep the connection alive as well.

> 

Reviewer: Can we add a step here to let us (the Community Team) know how the event went? We'd love to have better data on *actual* attendance and not rely on Meetup.com...

Reviewer: Thinking on this (and on the forum) does it make sense to create a private meetup organizers matrix room? @cybette?

Reviewer 2: Yes I like the idea of a private matrix room for organizers to replace the numerous DMs I get from individual organizers :)

Author: "The private Matrix room helps to keep the connection alive and continue with the conversation." - Updated with this line.

# Resources

Here are some example emails, social media posts to help the organizers with the different set communication while planning and excuting the meetup. The community members are welcome to share their ideas and contribute to the resources in the by sending a Pull Request [`ansible-community/meetup` Github repository](https://github.com/ansible-community/meetup). 

## Email Templates

### Speaker Outreach for the first time

```
Hello,
 
I am {name} , co-organizer at Ansible {place/city of the meetup} Meetup [0].

Ansible is an open-source community project sponsored by Red Hat [1].
It is the simplest way to automate IT. Ansible delivers simple IT automation that ends repetitive tasks and frees up DevOps teams for more strategic work.

> Ansible Community [2] is a gathering of a small group of Systems Engineers, DevOps, and SREs to discuss their automation journey focusing on Ansible.

Reviewer: The Ansible Community (as a whole) is anything but "small". I think that means something else to you?

Reviewer 2: I think it's probably Ansible _Location_ group (instead of Ansible Community) and a gathering of _local/regional_ System Engineers etc.

Author: Can you help me to correct sentence?

> Ansible {place/city of the meetup} Meetup is a platform for upstream and downstream to meet each other to share knowledge and Ansible stories.

Reviewer: upstream / downstream often mean subjective things. Perhaps we can use something like:

```suggestion
Ansible {place/city of the meetup} Meetup is a place for Ansible users and developers of all types to meet each other, share knowledge and swap Ansible stories.
```

Author: Updated.

{Share the reference you met or learned about the potential speaker and their work. Also, why would their talk be helpful for the event, group, and community? You can share some ideas for the talks with them.} 

Would you be interested in giving a talk or taking a workshop or hands-on session on {date} for the {name of the group}?

Please let me know if you need further information.

I will look forward to your answer.

Thank you,
{your_name}
```

### Follow-up email for the speaker

```
Hello {name},

This is {name} again.

Have you had a chance to look at the email I sent on {date}? I am attaching the last email here for your reference.

Please let me know if you need further information.

We are looking forward to your reply.

Thank you
{your_name}
```

### Email template once the speaker confirms for the talk

```
Hello,

Thank you for confirming the workshop/talk/hands-on. Your session will truly benefit the group.

Now to plan the session can you please provide the following information, whichever you are okay in sharing in with public.

1. Your full name or the name you prefer to be called by the community.

> 2. The pronoun you are comfortable with.

Reviewer: ```suggestion
2. The pronoun(s) you are comfortable with.
```

3. Your introduction, a short bio.
4. Title of the workshop/talk/hands-on
5. Short description of the workshop/talk/hands-on
6. The duration you are looking for the talk.
7. The name of your organization and if you are comfortable sharing them.
8. Are you okay if I tag you on Social media? Please share your social media handles (Mastodon, Linkedin, and Twitter) if you are comfortable.

> 9. Are you there in the Ansible Matrix? If not, then please follow. Can we share your Matrix username and social media handles in the meetup event page? It will be helpful for the attendees and broader community members to communicate with you.

Reviewer: ```suggestion
9. Are you on [Matrix](https://matrix.to/#/#space:ansible.com)? If not, then please consider signing up.
10. Can we share your Matrix username and social media handles in the meetup event page? It will be helpful for the attendees and broader community members to communicate with you.
```

Author: updated.

Let us know what information you are okay to share. Also please specify the adminstrative stuff, requirements for your presentation or any other details you want us know or take care.

Please share the details above by {date}. We intend to announce the event on {date}.

Looking forward to your reply.

Thank you,
{your_name}
```

### Email template once the event is announced

```
Hello {name},

We have announced the event. Please review the meetup page [0] and let me know if you want anything edited.

Feel free to promote the event on social media, and you can tag us :). The relevant social media handles are the following :

- [LinkedIn group](https://www.linkedin.com/groups/4648083/)
- [LinkedIn Ansible](https://www.linkedin.com/company/ansible/)
- [Mastodon (over Fostodon instance)](https://fosstodon.org/@ansible)
- [Twitter](https://twitter.com/ansible)

See you at the event.

Thank you,
{your_name}
```

### Email template for venue scout for the first time

```
Hello,

I am {name} , co-organizer at Ansible {place/city of the meetup} Meetup [0].

> Ansible is an open-source community project sponsored by Red Hat [1]. It is the simplest way to automate IT. Ansible delivers simple IT automation that ends repetitive tasks and frees up DevOps teams for more strategic work. Ansible Community [2] is a gathering of a small group of Systems Engineers, DevOps, and SREs to discuss their automation journey focusing on Ansible. Ansible {place/city of the meetup} Meetup is a platform for upstream and downstream to meet each other to share knowledge and Ansible stories. We meet every month/once in two months to talk about the above said topics and ideas.

Reviewer: same comments as above for upstream/downstream.

Author: Updated.

Please let me know if you need further information.

We are looking forward to your reply.

Thank you
{your_name}
```

### Email template for after-venue confirmation

```
Hello,

Thank you for confirming the venue for our Ansible {place/city of the meetup} meetup on {date}.

I will be visiting the venue before the event, so we are better prepared on the day of the event :). Please suggest a few suitable dates and times for the visit. Also, please confirm if there is any security requirement.

We will announce the event by the end of the week/shortly. Please let us know the following things by {specify a date}.
    - exact address of the venue
    - if there is any landmark near the venue (to help to reach the venue easily)
    - security requirement for entrance
    - the organization logo for the promotion
    - social media handles for promotion and the event page

We are looking forward to your reply.

Thank you
{name}
```

### Email template for reaching out Sponsor for the first time

```
Hello,

I am {name} , co-organizer at Ansible {place/city of the meetup} Meetup [0].

> Ansible is an open-source community project sponsored by Red Hat [1]. It is the simplest way to automate IT. Ansible delivers simple IT automation that ends repetitive tasks and frees up DevOps teams for more strategic work. Ansible Community [2] is a gathering of a small group of Systems Engineers, DevOps, and SREs to discuss their automation journey focusing on Ansible. Ansible {place/city of the meetup} meetup is a platform for upstream and downstream to meet each other to share knowledge and Ansible stories. We meet every month/once in two months to talk about the above said topics and ideas.

Reviewer: same again for upstream/downstream

Author: Updated.

```

```
Make your life easier with automation. Learn how to convert Ansible playbooks to Ansible roles with our speaker at the Ansible {place} meetup. Join us in the adventure on {date}, {time} {time zone}. #opensource #community
```

```
Ansible {name} Meetup group is rebooting its journey on {date} at {venue}, {time} {time zone}. Learn about the Ansible Ecosystem and Automation with AWX from our Ansible Upstream. RSVP now [here](link)
```

```
It is time for us at Ansible Community {place}, to meet again before we break for holiday. We will be listening to a journey of a Network admin and how he automated his work with Ansible roles. Then we will have a session to learn how to test roles with Molecule. Join us on {date} at {place}, {time} {time zone}. Register yourself on the event page.
```

### Promotion during the event : post templates

```
It is always so nice to see people writing their first Ansible Playbook, {ansible meetup group name} at {place}.
```

```
We have started with our {group_name} meetup with a full house. Now is the time to automate things with Ansible.  #community #opensource
```

### Logo to use 

> https://www.ansible.com/logos

Reviewer: Only some of these are for community use. @cybette can you advise here?

Reviewer 2: Mainy the use of the Mango one is discouraged (ok as an accent colour) so we can possibly remove that from the page. In addition, they should follow the guidelines and get approval for any designs, as "Any use of the Ansible logos and marks without Ansible approval is prohibited." 

So might be good to call to attention the Usage Guidelines on that page.
